.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: rust
 rust:
-  - 1.12.0
```

Comment:
I understand why we can no longer use Rust 1.12 (`serde_derive` requires Rust 1.15 IIRC), but there's a reason why a specific version is tested in Travis: to ensure we don't *accidentally* bump the minimum Rust version required to build docopt. For this reason, please put the minimum version back. I think Rust 1.15 would be suitable.